src/axolotl/integrations/llmcompressor_sft/args.py

```diff
@@ -0,0 +1,31 @@
+"""
+LLMCompressor and Sparse Finetuning config models.
+"""
+
+from typing import Any
+
+from pydantic import BaseModel, ConfigDict, Field
+from typing_extensions import Annotated
+
+
+class SFTArgs(BaseModel):
+    """Sparse Finetuning config for LLMCompressor."""
+
+    # Typing for recipe is set to Any due to:
+    # https://github.com/vllm-project/llm-compressor/issues/1319
+    recipe: Annotated[Any, Field(description="The recipe containing the compression algorithms and hyperparameters to apply.")]
+
+    model_config = ConfigDict(
```

Comment:
What is this? I noticed it's not specified in your example YAML config and it's vaguely named / typed.